.github/workflows/dev.yaml

```diff
@@ -31,6 +97,7 @@ jobs:
           push: true
           build-args: dev=true
           tags: |
-            gsscogs/pmdutils:dev
+            gsscogs/pmdutils:latest
```

Comment:
Could you create an issue so we update the [csvw2pmd pipeline](https://github.com/GSS-Cogs/pmd-jenkins-library/blob/csvwlib-build/vars/csvw2pmd.groovy#L22) in pmd-jenkins-library to make use of the `latest` image rather than the `dev` image which we're currently using.